src/components/card-details/index.tsx

```diff
@@ -1,29 +1,37 @@
 import React from 'react';
 import './card-details.css';
-import Dropdown from '../dropdown/index';
+import { openModal } from '../../store/reducers/modalReducer';
+import { useAppDispatch } from '../../hooks/useAppDispatch';
 
 interface CardDetailsProps {
   label?: string;
-  dropdown: DropdownProps[];
   color?: 'white' | 'gray';
-}
-
-interface DropdownProps {
-  name: string;
-  click: () => void;
+  title?: string;
+  cardItemId: number;
 }
 
 const CardDetails: React.FC<CardDetailsProps> = ({
   label,
-  dropdown,
   color = 'white',
+  title,
+  cardItemId,
 }) => {
+  const dispatch = useAppDispatch();
+
   return (
-    <div className={`styled-card-detail colour-${color}`}>
+    <div
+      className={`styled-card-detail colour-${color}`}
+      onClick={() =>
+        dispatch(
+          openModal({
+            component: 'update-card',
+            title: 'Update Card',
+            meta: { id: cardItemId, name: label, title: title },
+          })
```

Comment:
Bu CardDetails props title ve cardItemId koyduğunda sen dediğinde gibi gereksiz yanı, yukarı açıkladığım gibi yeni bir props onClickRemove koyarsan gereksiz olmayacak. Önemli olan minify için

Reply:
aynı sekilde burada da minify yapmak için function mı olmalı anlamadım

Reply:
düzeltmiştin görüyorum